code-challenges.js

```diff
@@ -18,9 +23,14 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
-
-
-
+ const cubed = (array)  => {
+   var sum = array.reduce(function(a,b) {
+       return a + b
+   }, 0 )
+   return sum ** 3
+ }
+console.log(cubed(cubeAndSum1))
+console.log(cubed(cubeAndSum2))
```

Comment:
I am tracking your logic, but it looks like you are adding up the numbers and then cubing them so you are not getting the correct output.